dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -18,8 +18,9 @@
     = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :is_k1, description: "Is Pre-Reader level"}
     = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :disable_variable_editing, description: "Disable editing of variable names and creation of new variables"}
     = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :show_type_hints, description: "Show a small visual hint for what type of block a connection accepts"}
-
-  = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :use_modal_function_editor, description: "Use modal function editor"}
+    -# Google Blockly does not currently support the modal function editor
+  - unless (@level.is_a?(Dancelab) || @level.is_a?(Poetry))
```

Comment:
Not something that needs to be handled in this PR but would it makes sense to have a method to check this on each level? Something like `@level.supports_google_blockly?`?

Reply:
Yes, I think this is possible: https://github.com/code-dot-org/code-dot-org/pull/48857#discussion_r1006253039

Right now that logic wouldn't quite get us where we need as Dance isn't on Google Blockly _yet_. The current plan is to switch to that method once we flip Dance to Google Blockly. This is being tracked in the draft PR that was started for the actual flip.